### ISSUE TITLE
Toolbar icons are missing.

### ISSUE THREAD

**Issue by kalzEOS** [GianniLab/Mailspring-11-theme#2]
All the toolbar icons are missing, and are replaced with some text. See screenshot, please. I've cloned the repo, installed the theme from within mailspring, and it told me that it was installed. Applied the theme and the icons are missing. Tried rebooting, but that didn't fix it. Are there any additional things I need to do beside installing the theme?
![sss](https://user-images.githubusercontent.com/49320606/160203315-b3195e2b-1c69-4fe7-92e1-ff22cdc68551.png)

Mailspring version: 1.9.2
Operating System: Manjaro Linux
KDE Plasma Version: 5.24.3
KDE Frameworks Version: 5.91.0
Qt Version: 5.15.3
Kernel Version: 5.15.28-1-MANJARO (64-bit)
Graphics Platform: X11
Processors: 8 × Intel® Core™ i7-8550U CPU @ 1.80GHz
Memory: 15.4 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics 620

**Comment by GianniLab** [comment]
Hi @KarkanAlzwayed ,
i don't have linux, so i didn't foresee it. To fix this you have 2 options.
Option 1:
Use the original Mailspring icons. Go to the installation folder  `~/11-light/styles/ `
open the file `index.less` and comment or delete the line with `@import custom-icons` ;
Option 2:
Still in the same folder, open the file `custom-icon.less`  and replace the path where the theme is located.
Original:
`@baseurl:" ../../../../../../Roaming/Mailspring/packages/11-light/img`  .
Example of your custom path:
`@baseurl:" ~/.config/Mailspring/packages/11-light / img`  .

**Comment by kalzEOS** [comment]
@GianniLab The second option actually worked, but not the way you did it. I had to add the full path to the theme before installing it from mailspring. I used this path `@baseurl:"/home/kalzi/.config/Mailspring/packages/11-light/img";` So, thanks for that.
I do have a small question if you don't mind, is this icon in the below screenshot supposed to be this big?
Edit: Nevermind, I just clicked it, and then it fixed itself. Theme is fantastic, btw. Thank you.
![ashdf](https://user-images.githubusercontent.com/49320606/160262463-c6df20c2-fbd2-4ad5-89ab-624e4b45b084.png)
